cmd/ddev/cmd/commands.go

```diff
@@ -18,8 +20,30 @@ import (
 	"github.com/ddev/ddev/pkg/util"
 	"github.com/mattn/go-isatty"
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
 )
 
+type annotations = struct {
+	// @TODO maybe some should not be omit empty
+	// @TODO maybe allow or even enforce capital letter starts
+	// @TODO check for any missing annotations and make sure we parse and use them
+	Aliases           []string `yaml:"aliases,omitempty"`
+	AutocompleteTerms []string `yaml:"autocompleteTerms,omitempty"`
+	CanRunGlobally    bool     `yaml:"canRunGlobally,omitempty"`
+	DbTypes           []string `yaml:"dbTypes,omitempty"`
+	Description       string   `yaml:"description,omitempty"`
+	DisableFlags      bool     `yaml:"-"`
+	Example           string   `yaml:"example,omitempty"`
+	ExecRaw           bool     `yaml:"execRaw,omitempty"`
+	Flags             Flags    `yaml:"flags,omitempty"`
+	HostBinaryExists  string   `yaml:"hostBinaryExists,omitempty"`
```

Comment:
I think it's better to make it an array for readability:

```suggestion
	HostBinaryExists  []string `yaml:"hostBinaryExists,omitempty"`
```